.github/workflows/auto-merge.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright The Enterprise Contract Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+name: 'Auto merge'
+'on':
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  auto-merge:
+    uses: enterprise-contract/github-workflows/.github/workflows/auto-merge.yaml@main
```

Comment:
Why not use the `.github` repo since that already has shared workflows?

Reply:
hmm we should probably move the other workflows there then. Having shared workflows in two separate repos is confusing.

Reply:
The [error](https://github.com/zregvart-test/testbed/actions/runs/6378405326) I was getting for reference https://github.com/zregvart-test/.github/blob/main/workflows/auto-merge.yaml

![Screenshot 2023-10-02 at 16-31-40 Auto merge · zregvart-test_testbed@b64770c](https://github.com/enterprise-contract/ec-cli/assets/1306050/6638735c-7ad8-4701-b22b-a74db640188b)

Reply:
Could it be because it's across orgs/namespaces?

Reply:
oy... let's file a story to move the other shared workflows to the new repo then?